test/tests/functions/parsed/vals_error.i

```diff
@@ -0,0 +1,29 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
+
+[Functions]
+  [./left_bc]
+    type = ParsedFunction
+    value = s
+    vals = invalid
+    vars = s
```

Comment:
Obviously small...but `vars=dummy` or something like that might make more sense?